docs/tests/integration/test_vulnerability_detector/test_general_settings/test_general_settings_min_full_scan_interval.md

```diff
@@ -0,0 +1,59 @@
+# Test general settings min full scan interval
+
+The tests will modify the value of `min_full_scan_interval` tag in `ossec.conf`, 
+set different intervals and check the result in `ossec.log`.
+
+## General info
+
+|Tier | Number of tests | Time spent| Test file |
+|:--:|:--:|:--:|:--:|
+| 0 | 4 | 6m27s | test_general_settings_min_full_scan_interval.py
+
+## Test logic
+
+For different time units:
+
+```
+['s','m','h','d']
+```
+
+and the time value:
+
+```
+[120]
+```
+
+The test will check in `ossec.log` if the following message appears:
+
+```
+<vulnerability_detector_prefix> A full scan will be run on agent...
+```
+
+## Checks
+
+- [x] Vulnerability Detector does not launch a full scan of the agent even if the CVEs database is 
```

Comment:
```suggestion
- [x] Vulnerability Detector does not start a full scan of the agent even if the CVEs database is 
```

Reply:
Done in: https://github.com/wazuh/wazuh-qa/pull/1369/commits/7438aaa4e055a855a5bd2cbd1fff70d33a42c43f